src/NUnitTestAdapter/TestingPlatformAdapter/NUnitExtension.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+
+using Microsoft.Testing.Platform.Extensions;
+
+namespace NUnit.VisualStudio.TestAdapter.TestingPlatformAdapter
+{
+    internal sealed class NUnitExtension : IExtension
```

Comment:
Extension points in `Microsoft.Testing.Platform` uses this base interface that provides some identity about the extension. This is used to ease diagnostic for end users.